test/unit/vl-core.test.html

```diff
@@ -75,6 +98,17 @@
         });
       });
 
+      test('wanneer er een attribuut met of zonder prefix uit de observed attributes lijst wijzigt, zal bij het ontbreken van een changed callback de prefixed changed callback functie aangeroepen worden', () => {
```

Comment:
We kunnen misschien de dubbele functies extracten en de fixture meegeven per test suite? Op die manier weten we dat we telkens voor elk type element exact hetzelfde testen.

Reply:
Ik vind duplicatie op test niveau nooit erg, elke test kan dan volledig in isolatie gelezen worden. Ik denk ook dat het perfect mogelijk is dat er een breaking change kan gebeuren op zowel native als niet native elementen, dat zou ik dan ook zo in de testen willen lezen. Een falende test die alles test, zegt niets over de bug en geeft dan weer complexiteit in het debuggen.

Reply:
ge kunt het wel in een functie steken en de 2 testen houden he ...

Reply:
Yes, dat was mijn bedoeling. Je houdt uw 2 testen maar je steekt de inhoud in een functie die gebruikt wordt door die testen. 
En ja, in de toekomst is het misschien ineens niet meer hetzelfde, maar nu wel nog. Sterker, het erft van elkaar over dus er is al een harde koppeling momenteel.